src/coreaudiokit.cs

```diff
@@ -28,7 +28,7 @@ namespace XamCore.CoreAudioKit {
 #if XAMCORE_2_0 || !MONOMAC
 	[iOS (9,0)][Mac (10,11, onlyOn64 : true)]
 	[BaseType (typeof(AUViewControllerBase))]
-	interface AUViewController : NSExtensionRequestHandling {
+	interface AUViewController {
```

Comment:
@VincentDondain: This is a breaking change:

![screenshot 2018-01-29 10 39 14](https://user-images.githubusercontent.com/249268/35503554-c5451872-04e0-11e8-92e8-3a060e7b978e.png)

Reply:
@rolfbjarne It's `UIViewController` that implements `NSExtensionRequestHandling` now (as part of [this PR](https://github.com/xamarin/xamarin-macios/pull/3187/files#diff-1ca9e8e46629c0f8806cee2c8be4aad3R13599)).

Then based on https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#types:

**Allowed**
> Removing an interface implementation from a type when the interface is already implemented lower in the hierarchy

Reply:
Also I asked about this exact case and you answered me :P

https://xamarinhq.slack.com/archives/C03CCJNR7/p1515786537000437

Vincent:

> I’m getting a breaking change from API diff but I don’t understand how come it’s a breaking change
> I move a protocol from a subclass to its parent. So the generated method from the protocol is removed from the subclass (that’s what’s reported as a breaking change) but that method is now on the base class…
> so even if a subclass of a subclass was overriding that virtual method, it should still be fine
> https://gist.github.com/VincentDondain/dbd1efe6976e748a2d72df10775a4524#file-new-diff-L189-L213 and https://gist.github.com/VincentDondain/dbd1efe6976e748a2d72df10775a4524#file-new-diff-L599-L623

Rolf:

> You’re assuming apidiff is infallible :slightly_smiling_face:
> Moving members to a base class is one case where it will say it’s a breaking change when it isn’t

Reply:
@VincentDondain heh, ok, hopefully I'll remember the next time I run into this :smile: 👍